t/10-jobs.t

```diff
@@ -662,7 +662,8 @@ subtest 'job is marked as linked if accessed from recognized referal' => sub {
     $job->update;
     $linked = job_is_linked($job);
     is($linked, 0, 'new job is not linked');
-    $t->get_ok('/tests/' . $job->id . '/modules/' . $module->id . '/steps/1' => {Referer => 'http://test.referer.info'})
+    $t->get_ok(
+        '/tests/' . $job->id . '/modules/' . $module->id . '/steps/1' => {Referer => 'http://test.referer.info/foobar'})
```

Comment:
To prevent the line split I suggest to pull out the string 'http://test.referer.info/foobar' into a variable which you can reuse